src/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -142,4 +142,32 @@
     <protected>false</protected>
     <shortDescription>Abandon Request</shortDescription>
   </labels>
+  <labels>
+    <fullName>cookbook_LogIn</fullName>
+    <value>Log in</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Log in</shortDescription>
+  </labels>
+  <labels>
+    <fullName>cookbook_DescribeProblem</fullName>
+    <value>Describe problem</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Describe problem</shortDescription>
+  </labels>
+  <labels>
+    <fullName>cookbook_ProvideDetails</fullName>
+    <value>Provide details</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Provide details</shortDescription>
+  </labels>
+  <labels>
+    <fullName>cookbook_ReviewHelp</fullName>
+    <value>Review help</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Review help</shortDescription>
```

Comment:
Seems like the steps names are a bit weird.
@acarpentier01 perhaps you could review them please?

Reply:
@mmitiche Is "Review help" supposed to be "Request help"? The other step names look fine to me.

Reply:
This is the title of the step where we propose the user some document suggestion where hr could potentially find a solution to his case. you can find a screenshot  of this step bellow.
@acarpentier01  Maybe "Review help resources" would be good for this step? what do u think? I'll be fine with any other suggestion you may propose.
<img width="1335" alt="review-step" src="https://user-images.githubusercontent.com/86681870/151005869-7c6f7d31-c8b7-4166-a62d-f3b2214eeb47.png">

Reply:
@mmitiche I like that suggestion! If it's too long, you could also go with "Review resources".